# Mark these as dependent joints, but handle this dependency yourself,
# instead of letting the joint_state_publisher doing it, since it can't handle
# all the dependency.
# Put these here so they don't show up on the gui, although the results
# are just ignored.
dependent_joints:
    upper_to_lower_top_1: {parent: base_to_upper_1, factor: 0 }
    upper_to_lower_top_2: {parent: base_to_upper_2, factor: 0 }
    upper_to_lower_top_3: {parent: base_to_upper_3, factor: 0 }
    lower_top_to_lower_left_1: {parent: base_to_upper_1, factor: 0 }
    lower_top_to_lower_left_2: {parent: base_to_upper_2, factor: 0 }
    lower_top_to_lower_left_3: {parent: base_to_upper_3, factor: 0 }
    lower_top_to_lower_right_1: {parent: base_to_upper_1, factor: 0 }
    lower_top_to_lower_right_2: {parent: base_to_upper_2, factor: 0 }
    lower_top_to_lower_right_3: {parent: base_to_upper_3, factor: 0 }
    lower_left_to_lower_bot_1: {parent: base_to_upper_1, factor: 0 }
    lower_left_to_lower_bot_2: {parent: base_to_upper_2, factor: 0 }
    lower_left_to_lower_bot_3: {parent: base_to_upper_3, factor: 0 }
    upper_to_pseudo_elbow_offset_1: {parent: base_to_upper_1, factor: 0 }
    upper_to_pseudo_elbow_offset_2: {parent: base_to_upper_2, factor: 0 }
    upper_to_pseudo_elbow_offset_3: {parent: base_to_upper_3, factor: 0 }
    pseudo_elbow_offset_to_pseudo_elbow_1: {parent: base_to_upper_1, factor: 0 }
    pseudo_elbow_offset_to_pseudo_elbow_2: {parent: base_to_upper_2, factor: 0 }
    pseudo_elbow_offset_to_pseudo_elbow_3: {parent: base_to_upper_3, factor: 0 }
    pseudo_elbow_to_virtual_lower_1: {parent: base_to_upper_1, factor: 0 }
    pseudo_elbow_to_virtual_lower_2: {parent: base_to_upper_2, factor: 0 }
    pseudo_elbow_to_virtual_lower_3: {parent: base_to_upper_3, factor: 0 }
    lower_bot_1_to_virtual_end_effector_offset: {parent: base_to_upper_3, factor: 0 }
